src/main/java/nl/thehyve/ocdu/factories/EventDataFactory.java

```diff
@@ -133,11 +135,9 @@ protected void setValue(String[] row, Map<String, Integer> columnsIndex, String
                 row.add("");//Start Time
                 row.add("");//End Date
```

Comment:
For the future: One way we could address some of the issues users face with filling in tab delimited files is by putting placeholders (with incorrect values so that they are not accidentally submitted)